src/components/card/index.tsx

```diff
@@ -30,10 +50,16 @@ const Card: React.FunctionComponent<CardProps> = props => {
 
 	const body = <div className={`${prefixCls}-body`}>{children}</div>
 
+	const actionDom =
+		actions && actions.length ? (
+			<ul className={`${prefixCls}-actions`}>{getAction(actions)}</ul>
```

Comment:
May be this come from the source code but I think it is unnecessary to pass `actions` as an argument here while we could get it from props in `getAction` method